src/mobile/ios/IOS_DEPLOY_GUIDE.md

```
@@ -0,0 +1,223 @@
+---
+
+## Building the App
+
+### Option A: Command Line (recommended)
+```bash
+# From the mobile/ directory
+flutter build ipa --release
+```
+
+The `.ipa` file will be at:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)